typescript/sdk/src/index.ts

```diff
@@ -1,6 +1,24 @@
-export { AbacusApp } from './app';
```

Comment:
Instead of index files in each subfolder, there's now just this one master index which will determine exactly what the SDK exports

Reply:
nice